spec/controllers/bookmarks_controller_spec.rb

```diff
@@ -10,6 +11,10 @@ def it_redirects_to_user_login
 
   describe 'new' do
     context 'without javascript' do
+      let(:chapteredwork) { create(:work) }
```

Comment:
Could you add an underscore between words to bring this more in line with our variable naming conventions?

Reply:
Will do.